### PR TITLE
Rename `master` to `main`

### PR DESCRIPTION
**Pull request by pitag-ha**
This commit adapts the user manual and the CIs to rename `master` to `main`.

### CODE REVIEW

**Comment by pitag-ha** [comment]
Of course, the AppVeyor-CI can only pass once `master` is actually renamed to `main`. Once this PR is approved, I'll first rename the branch, then wait for the CI to pass and then merge.

**Review by NathanReb — APPROVED** [review]

**Pull request merged**
